lib/danger/commands/init.rb

```diff
@@ -244,7 +244,7 @@ def unsure_token
     end
 
     def note_about_clicking_links
-      ui.say "Note: Holding cmd ( ⌘ ) and #{ENV['ITERM_SESSION_ID'] ? '' : 'double '}clicking a link will open it in your browser."
```

Comment:
Looks like it - http://ruby-doc.org/stdlib-1.9.2/libdoc/rubygems/rdoc/Gem/Platform.html